ui-v2/tests/integration/services/services-test.js

```diff
@@ -58,16 +59,22 @@ skip('findBySlug returns the correct data for item endpoint', function(assert) {
       assert.deepEqual(
         actual,
         expected(function(payload) {
-          payload = { Nodes: payload };
+          // TODO: So this tree is all 'wrong', it's not having any major impact
+          // this this tree needs revisting to something that makes more sense
+          payload = Object.assign(
+            {},
+            { Nodes: payload },
+            {
+              Datacenter: dc,
+              uid: `["${dc}","${id}"]`,
+            }
+          );
```

Comment:
So this tree is not very nice, and wasn't what I was expecting. I've tested for it as this PR is to refactor what is there without introducing any regressions. I've added a note and a skipped test to remind me to come back here at a later date.